horace_core/sqw/file_io/@faccess_sqw_v4/get_sqw.m

```diff
@@ -68,18 +68,32 @@
 [obj,sqw_skel] = obj.get_all_blocks(sqw_skel,'ignore_blocks',skip_blocks);
 
 if ~(opts.head || opts.his)
-    detpar = sqw_skel.detpar;
-    if ~isempty(detpar) && ~isempty(detpar.group)
-        detpar = IX_detector_array(detpar);
-    else
-        detpar = IX_detector_array();
-    end
-    %detpar = repmat(detpar,numel(sqw_skel.experiment_info.expdata),1);
+    % detpar-independent inputs
     sqw_skel.data = DnDBase.dnd(sqw_skel.data.metadata,sqw_skel.data.nd_data);
     sqw_skel.experiment_info = Experiment([],sqw_skel.experiment_info.instruments, ...
         sqw_skel.experiment_info.samples,sqw_skel.experiment_info.expdata);
-    sqw_skel.experiment_info.detector_arrays = ...
-        sqw_skel.experiment_info.detector_arrays.add_copies_(detpar, numel(sqw_skel.experiment_info.expdata));
+    
+    % detpar inputs
+    detpar = sqw_skel.detpar; % $DET
```

Comment:
What does this comment mean?

Reply:
it's a marker for detpar usage, can be removed